openedx/core/djangoapps/user_api/tests/test_constants.py

```diff
@@ -1,10 +1,31 @@
 # -*- coding: utf-8 -*-
 """Constants used in the test suite. """
 SORTED_COUNTRIES = [
+    (u"DZ", u"Algeria"),
```

Comment:
Why this change is needed?

We have the `COUNTRIES_FIRST` setting for that:

```
$ git grep -A6 -i countries_first
lms/envs/common.py:COUNTRIES_FIRST = [
lms/envs/common.py-    'DZ',
lms/envs/common.py-    'BH',
lms/envs/common.py-    'KM',
lms/envs/common.py-    'DJ',
lms/envs/common.py-    'EG',
lms/envs/common.py-    'IQ',
--
lms/envs/common.py:COUNTRIES_FIRST_SORT = True  # Since the order depends on the name after translation
lms/envs/common.py-
lms/envs/common.py:COUNTRIES_FIRST_BREAK = '--'
lms/envs/common.py-
lms/envs/common.py-# which access.py permission name to check in order to determine if a course is visible in
lms/envs/common.py-# the course catalog. We default this to the legacy permission 'see_exists'.
lms/envs/common.py-COURSE_CATALOG_VISIBILITY_PERMISSION = 'see_exists'
lms/envs/common.py-
lms/envs/common.py-# which access.py permission name to check in order to determine if a course about page is
--
openedx/core/djangoapps/user_api/tests/test_views.py:        self.assertTrue(settings.COUNTRIES_FIRST_SORT, 'Should re-sort the COUNTRIES_FIRST for each language')
--
openedx/core/djangoapps/user_api/tests/test_views.py:                if country_code in settings.COUNTRIES_FIRST
--
openedx/core/djangoapps/user_api/tests/test_views.py:                if country_code not in settings.COUNTRIES_FIRST
```

Reply:
@OmarEdraak the problem is in `COUNTRIES_FIRST_SORT = True` . It is causing two tests to fail because the result is already sorted; and therefore, not equal to `SORTED_COUNTRIES`
`test_register_form_third_party_auth_running_google`
`test_registration_form_country`

Should we switch the default value in test environment, patch it here, or keep like this? What do you suggest?